cli/providers/providers.go

```diff
@@ -139,11 +139,6 @@ func attachProvidersToCmd(existing providers.Providers, cmd *Command) {
 		for j := range provider.Connectors {
 			conn := provider.Connectors[j]
 			attachConnectorCmd(provider.Provider, &conn, cmd)
-			for k := range conn.Aliases {
```

Comment:
This was needed, to get aliases work with plugins: https://github.com/mondoohq/cnquery/pull/1696/commits/24ee6f5178d9f8dd83dd865fcdd7cf9ca1d65587

Need to double check, whether this still works.

Reply:
Seems, this is no longer needed. Gave it a try with filesystem/fs and kubernetes/k8s. Both work.